Tests/MongoSwiftTests/MongoDatabaseTests.swift

```diff
@@ -39,7 +39,10 @@ final class MongoDatabaseTests: MongoSwiftTestCase {
 
         // error code 59: CommandNotFound
         expect(try db.runCommand(["asdfsadf": ObjectId()]))
-                .to(throwError(ServerError.commandError(code: 59, message: "", errorLabels: nil)))
+                .to(throwError(ServerError.commandError(code: 59,
+                                                        codeName: "CommandNotFound",
+                                                        message: "",
```

Comment:
When pattern matching, you can omit them with underscores. Here we're actually constructing an enum case, so we need to specify them all.

This PR did actually bring it to my attention that because we're using enum types for our errors, they're incredibly brittle. Because users will have to pattern match on them, even pure additive changes require a major version bump. I filed [SWIFT-502](https://jira.mongodb.org/browse/SWIFT-502) about this so that we could talk about it in triage.